### ISSUE TITLE
Matplotlib Style

### ISSUE THREAD

**Issue by ale-dg** [enkia/tokyo-night-vscode-theme#75]
Hi, 

This is not an issue per-se, but rather a contribution. I have created a style for Python's Matplotlib using the colours from your theme for VSCode. Nothing too fancy, just a bit of theming in case someone is interested in it. However, I am not sure how should I publish it, meaning if I should build a wheel and letting you know about it, or uploading the stylesheet so anyone can copy it and include it in their library. Would you let me know what could be more convenient, please?

See a screenshot below, for reference on what I mean.

Thanks

Best
![example_tokyo_night](https://github.com/enkia/tokyo-night-vscode-theme/assets/106413328/9e39bb06-afaa-4103-bbab-b06447537c0b)

**Comment by enkia** [comment]
Would a public GitHub snippet that I could link to in the README work?

**Comment by dyzastro** [comment]
> Owner

Yes, including a link to a public GitHub snippet in your README file is a great way to share code or configuration examples with users or collaborators i believe

**Comment by enkia** [comment]
Do you have one for me to link to?
